### DISCUSSION TITLE
github actions How to fill in “known-hosts”

### DISCUSSION THREAD

**Discussion by tlerbao** [deployphp/deployer#3286] [Help needed]
How should known-hosts be filled? What is the format?
grateful!

<img width="788" alt="image" src="https://user-images.githubusercontent.com/20442232/188292804-96282e1f-a685-4e9b-ab46-d0d119804b37.png">

**Comment by antonmedv** [comment]
As in your local know_hosts file:

    fs.appendFileSync(`${sshHomeDir}/known_hosts`, knownHosts)

**Reply by tlerbao** [reply]
I don't quite understand. Does this call my PC's known_hosts or the known_hosts of the server I'm deploying?

**Reply by antonmedv** [reply]
Of a VM GitHub action is running.
